pkgs/by-name/cl/clight-gui/package.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, libsForQt5
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "clight-gui";
+  version = "unstable-2023-02-21";
+
+  src = fetchFromGitHub {
+    owner = "nullobsi";
+    repo = "clight-gui";
+    rev = "29e7216bfcc68135350a695ce446134bcb0463a6";
+    hash = "sha256-U4vaMwnVDZnYLc+K3/yD81Q1vyBL8uSrrhOHbjbox5U=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/src";
+
+  nativeBuildInputs = [
+    cmake
+    libsForQt5.wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libsForQt5.qtbase
+    libsForQt5.qtcharts
+  ];
+
+  meta = with lib; {
```

Comment:
Using `finalAttrs` with `with lib;` can sometimes be problematic, because `with` doesn't shadow. Recommend to avoid both if you opt to use `finalAttrs`.